client/views/windows/connect.tpl

```diff
@@ -45,8 +45,8 @@
 			</div>
 			<div class="col-sm-9 col-sm-offset-3">
 				<label class="tls">
-					<input type="checkbox" name="allowUnauthorized" {{#unless defaults.rejectUnauthorized}}checked{{/unless}} {{#if lockNetwork}}disabled{{/if}}>
-					Allow untrusted certificates
+					<input type="checkbox" name="rejectUnauthorized" {{#if defaults.rejectUnauthorized}}checked{{/if}} {{#if lockNetwork}}disabled{{/if}}>
+					Accept only trusted certificates
```

Comment:
I believe this should be `Accept trusted certificates only`, but you'd be a fool to listen to me 🇫🇷 :D
@McInkay, could you confirm?
Apart from that, I'm 👍 on the wording

Reply:
[When your only native English speaker is a Scot ...](https://www.youtube.com/watch?v=sAz_UvnUeuU)

I think `Accept trusted certificates only` would be alright. But we could also do `Only allow trusted certificates`.

Reply:
`Only allow trusted certificates` sounds good to me.